<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Synchronisation</class>
 <widget class="QWidget" name="Synchronisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synchronisation</string>
  </property>
  <widget class="QPushButton" name="pushButtonSynchroniser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Synchroniser</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>168</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutSelectionSmartphone">
    <item>
     <widget class="QLabel" name="labelSelectionSmartphone">
      <property name="text">
       <string>Liste des smartphones connect√©s :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxSelectionSmartphone"/>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDetails">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Details &gt;&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetLog">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
